content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
+```
+
+Assume you have a monorepo with the following structure:
+
+```
+.
+├── packages/
+│   ├── prisma/
+│   │   ├── client/ <-- Custom output location for the generated Prisma Client
+│   │   └── schema.prisma
+│   └── service/
+│       ├── pages/
+│       │   └── api/
+│       │       └── test.js
+│       ├── next.config.js
+│       └── package.json
+├── pnpm-workspaces.yaml
+├── package.json
+└── vercel.json
+```
+
+This error mentioned above occurs as a result of a bundling problem during Next.js's bundling process. The `schema.prisma` file is treated as an artifact that should be bundled along with the rest of the `/api/test` endpoint's bundle.
+
+This causes an issue as Prisma Client expects `schema.prisma` to be available at a specific location in relation to the bundled code.
+
+<Admonition type="info">
+
+For a more detailed explanation of exactly what is going wrong during the bundling process, please refer to [this issue](https://github.com/vercel/next.js/issues/46070) we opened in the Next.js GitHub repository.
+
+</Admonition>
+
+## Solution
+
+To solve this issue, you can use a custom Webpack plugin we created that correctly copies the files Prisma Client needs.
+
+To use this plugin, first install the package:
+
+```shell copy
+npm install -D @prisma/webpack-plugin
+```
+
+You can then import the plugin into your `next.config.js` file and use it as shown below:
```

Comment:
I suggest to make this more generic explanation, and then show a concrete example:
```suggestion
You can then import the plugin into your `next.config.js` file and use it in `config.plugins`. For example:
```
Many users will already have a quite full `next.config.js` so they benefit more from the instructions how to use it than from a copy-pastable example.